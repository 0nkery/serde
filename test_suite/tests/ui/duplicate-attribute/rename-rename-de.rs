#[macro_use]
extern crate serde_derive;

#[derive(Serialize)]
struct S {
    #[serde(rename = "x")]
    #[serde(rename(deserialize = "y"))]
    x: (),
}

fn main() {}
