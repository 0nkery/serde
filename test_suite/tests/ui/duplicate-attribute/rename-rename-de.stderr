error: duplicate serde attribute `rename`
 --> $DIR/rename-rename-de.rs:7:13
  |
7 |     #[serde(rename(deserialize = "y"))]
  |             ^^^^^^

error: aborting due to previous error

